src/modules/shared/layouts/AppLayout/AppLayout.tsx

```diff
@@ -67,7 +67,7 @@ import { scrollTo } from '@shared/utils/scroll-to-top';
 
 import packageJson from '../../../../../package.json';
 
-import { useGetMaintenanceAlerts } from 'modules/maintenance-alerts/hooks/get-maintenance-alerts';
+import { useGetActiveMaintenanceAlerts } from 'modules/maintenance-alerts/hooks/get-maintenance-alerts';
```

Comment:
Maybe add `@maintenance-alerts` to the paths in `tsconfig.json` and import it the same way we import all other files from modules ;)